popup.js

```diff
@@ -0,0 +1,87 @@
+var loginUrl = "http://localhost:1337/auth/signin/chromeExtension"
+var logoutUrl = "http://localhost:1337/auth/logout";
+
+// var loginUrl = "https://http://adminer.hiiir.com/auth/signin/chromeExtension"
+// var logoutUrl = "https://http://adminer.hiiir.com/auth/logout";
+
+function getCurrentTabUrl(callback) {
+  initialLoginButton();
+  initialLogoutButton();
+  callback();
+};
+
+function initialLoginButton(){
+  $("#login").click(function(){
+    var username = $("input[name='username']").val();
+    var password = $("input[name='password']").val();
+    var remember_me = $("input[name='remember_me']").val();
+
+    $.ajax({
+      method: "POST",
+      url: loginUrl,
+      data: {
+        username: username,
+        password : password ,
+        remember_me : remember_me
+      }
+    })
+      .success(function( msg ) {
```

Comment:
Deprecation Notice: The jqXHR.success(), jqXHR.error(), and jqXHR.complete() callbacks are removed as of jQuery 3.0. You can use jqXHR.done(), jqXHR.fail(), and jqXHR.always() instead.